src/plugins/chart_expressions/expression_tagcloud/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "../../../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
```

Comment:
I saw that this attribute has been removed from all tsconfig.json in master

Reply:
Done.